### PR TITLE
fix: use starts_at instead of created_at for session reservation & elapsed time

### PR DESCRIPTION
**Pull request by gahyuun**
### TL;DR

This PR resolves issue that session reservation time and session elapsed time were incorrectly displayed 

###  Why make this change?

session reservation time should mean session start time, not session creation time
session elapsed time should be  caculated by session start time,  not session creation time

### What changed?

session reservation time means session start time
session elapsed time is calculated based on the session start time and session terminate time
session elapsed time is displayed only after the session starts

### How to test?

1. set a session start time
2. start a session
3. check that reservation time is displayed correctly
4. check that elapsed time is displayed after the session starts and elapsed time is calculated correctly

### CODE REVIEW

**Review by ironAiken2 — CHANGES_REQUESTED** [review]
Elapsed time and date are not displayed correctly.

**Review by ironAiken2 — APPROVED** [review]
LGTM

**Review by agatha197 — APPROVED** [review]
LGTM
